.github/workflows/build.yml

```diff
@@ -22,11 +28,11 @@ jobs:
 
       - name: Run rubocop
         run: |
-          bundle exec rubocop --display-cop-names
+          bin/rubocop --display-cop-names
```

Comment:
I'm not really sure about this change, I think internally we always use bundle exec 🤔 
Any reason why wasn't it working for you?

Reply:
The both end up running in the context of Bundler. Partially, binstubs are a habit/ergonomics that I've grown accustomed to. But in terms of DX, the existence of various binstubs allows some tooling (think, Vim plugins, VSCode extensions, etc…) to make decisions about how to invoke tooling. For example, a `bin/rubocop` existing let's Vim's ALE plugin know that it should use Rubocop (and the right version of it by way of the binstub) rather than Standard.rb as the linter/fixer.

You can still use `bundle exec` even with a binstub existing. But the existence of a binstub both signals "this is tooling expected to be here, and be used by, this code base," and it enables certain developer tooling to work without additional configuration.

All of that said, I can pull this out to a separate PR if you wish. Thoughts?

Reply:
@unsign3d I went ahead and rebased the binstubs changes out. I'd still like to at least get them added, even if we continue to use `bundle exec` in the GitHub Action - just so local tooling works out of the box. But we can talk about that in another PR, as you've suggested.

Reply:
@stevenharman yeah, thank you, I feel that it would be easier to discuss in a different PR, I don't have strong feelings about either, but I want to try to push things as fast as I can. 
We can talk about it in a different PR, I will in the meantime try to educate myself more on the topic :)

Reply:
Sounds good. Once this PR is merged I'll start opening smaller ones for these other changes; they just all depend on this one, so they look larger than they really are. 😄